bindata/etcd/cluster-backup-job.yaml

```diff
@@ -49,7 +49,12 @@ spec:
               name: static-pod-dir
             - mountPath: /etc/kubernetes/cluster-backup
               name: etc-kubernetes-cluster-backup
+            - mountPath: /var/run/secrets/etcd-client
```

Comment:
i wonder how was it working before then ?

Reply:
it's coded into the script itself
https://github.com/openshift/cluster-etcd-operator/blob/master/bindata/etcd/cluster-backup.sh#L110-L112

Reply:
achsooo :D :D